Application/EdFi.Ods.Api.IntegrationTestHarness/Artifacts/MsSql/Data/Security/0020-Custom-View-Based-Auth-Metadata.xml

```diff
@@ -0,0 +1,236 @@
+<SecurityMetadata>
```

Comment:
Support for C#-based security metadata manipulation is being replaced with the declarative XML-based management (in combination with XSLT-based SQL script generation).